src/App.vue

```diff
@@ -527,6 +530,28 @@ export default class App extends Vue {
     })
   }
 
+  /** Opens Auth Web in a new tab with the survey query param. */
+  protected launchFilingSurvey (): void {
+    const iaSurveyId = sessionStorage.getItem('IA_SURVEY_ID')
+
+    // safety check
+    if (iaSurveyId) {
+      const url = `${sessionStorage.getItem('AUTH_WEB_URL')}?survey=${iaSurveyId}`
+      this.window.open(url, '_blank', 'noreferrer')
+    }
+  }
+
+  /** Called to save or delete "hide survey" cookie. */
+  protected updateSurveyCookie (doNotShow: boolean): void {
+    const domain = window.location.hostname
+    if (doNotShow) {
+      // hide survey for 30 days
+      Cookies.set(this.IA_SURVEY_KEY, 'do-not-show', { domain, expires: 30 })
+    } else {
+      Cookies.remove(this.IA_SURVEY_KEY, { domain })
+    }
+  }
```

Comment:
Sample cookies:

![image](https://user-images.githubusercontent.com/10002889/214181383-b2dc13fb-c76c-4f1a-8684-c4ee1ca76b28.png)

I'm expecting/hoping the Domain will be "dev.bcregistry.ca" or whatever. This is currently running locally.

The Expires field is a cookie feature: the cookie is auto-removed then the date/time is reached.

The other fields are default values and should be OK as-is.